apps/edxapp/vars/all/main.yml

```
@@ -72,6 +72,8 @@ edxapp_export_volume_size: 100Mi
 
 # -- volume names
 edxapp_export_volume_path: /edx/var/edxapp/export
+required_directories:
```

Should be prefixed by the app name?
```suggestion
edxapp_required_directories:
```

---

oh yes! So obvious I couldn't see it ^^